cf/rds.yaml

```diff
@@ -0,0 +1,118 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Set up RDS Aurora cluster [managed by github.com/hms-dbmi-cellenics/iac]
+
+Parameters:
+  Environment:
+    Type: String
+    Default: development
+    AllowedValues:
+      - development
+      - staging
+      - production
+    Description: The environment for which the cluster needs to be created.
+
+# --------------------------------------- Constants ---------------------------------------
+  DBPort:
+    Type: String
+    Default: "5432"
+
+  DBEngineVersion:
+    Type: Number
+    Default: 13.4
+
+  DBSecretPrefix:
+    Description: The prefix of the secret used to store master user and password, 
+      the missing suffix is the environment (and both are connected with -).
+    Type: String
+    Default: aurora
+
+Resources:
+  DBCluster:
+    Type: AWS::RDS::DBCluster
+    DeletionPolicy: Snapshot
+    UpdateReplacePolicy: Snapshot
+    Properties:
+      Engine: aurora-postgresql
+      EngineMode: provisioned
+      EngineVersion: !Ref DBEngineVersion
+      DBSubnetGroupName: !Ref DBSubnetGroup
+      DBClusterIdentifier: !Join [ "-", ["aurora-cluster", !Ref Environment] ]
+      DatabaseName: "aurora_db"
+      Port: !Ref DBPort
+      # Can't be deleted before first setting this to false, should be true when we are done with the basic setup of db
+      DeletionProtection: false
+      MasterUsername: !Join ["", ["{{resolve:secretsmanager:", !Join ['-', [!Ref DBSecretPrefix, !Ref Environment]], ":SecretString:username}}" ]]
+      MasterUserPassword: !Join ["", ["{{resolve:secretsmanager:", !Join ['-', [!Ref DBSecretPrefix, !Ref Environment]], ":SecretString:password}}" ]]
+      EnableIAMDatabaseAuthentication: true
+      BackupRetentionPeriod: 5
+      SourceRegion: !Sub '${AWS::Region}'
+      # Check if we want to change this
+      StorageEncrypted: false
+      VpcSecurityGroupIds: 
+        - !Ref RDSSecurityGroup
+
+  DBSubnetGroup:
+    Type: AWS::RDS::DBSubnetGroup
+    Properties:
+      DBSubnetGroupDescription: !Sub Biomage subnet group for ${Environment}.
+      DBSubnetGroupName: !Sub rds-subnet-group-${Environment}
+      SubnetIds:
+        Fn::Split: [',', Fn::ImportValue: !Sub "eksctl-biomage-${Environment}-cluster::SubnetsPrivate"]
+      Tags:
+        - Key: "Name"
+          Value: AuroraSubnetGroup
+
+  DBInstance1:
+    Type: AWS::RDS::DBInstance
+    Properties:
+      DBInstanceClass: db.t3.medium
```

Comment:
how do you know that this is the right instance?
Did you switch off that instance that we (I) started from the console for testing purposes?

Reply:
We started an ec2 instance for testing I think, I don't remember us starting an RDS cluster from the console (we looked over the options). There's also the hackathon instance which I deleted some time ago (it was t3.medium too).

Based on [this](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Concepts.DBInstanceClass.html), t3s are recommended for development purposes, which is the state that this will be in for now, so it seems like it could be an adequate initial value.

I think looking over this with marcell we had decided to go with this one as an initial value.

Reply:
sounds good!